pom.xml

```diff
@@ -0,0 +1,463 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<parent>
+		<groupId>io.github.zentol.flink</groupId>
+		<artifactId>flink-connector-parent</artifactId>
+		<version>1.0</version>
+	</parent>
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.apache.flink</groupId>
+	<artifactId>flink-connector-opensearch-parent</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<name>Flink : Connectors : Opensearch : Parent</name>
+	<packaging>pom</packaging>
+	<url>https://flink.apache.org</url>
+	<inceptionYear>2022</inceptionYear>
+
+	<licenses>
+		<license>
+			<name>The Apache Software License, Version 2.0</name>
+			<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<scm>
+		<url>https://github.com/apache/flink-connector-opensearch</url>
+		<connection>git@github.com:apache/flink-connector-opensearch.git</connection>
+		<developerConnection>
+			scm:git:https://gitbox.apache.org/repos/asf/flink-connector-opensearch.git
+		</developerConnection>
+	</scm>
+
+	<modules>
+		<module>flink-connector-opensearch</module>
+		<module>flink-connector-opensearch-e2e-tests</module>
```

Comment:
missing the sql connector module

Reply:
It is not missing, driven by profile [1] which is active by default. Reason: it comes from Flink itself [2]

[1] https://github.com/apache/flink-connector-opensearch/pull/1/files/41cc7bb681aede94c543c28cc5629802874aac70#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R385
[2] https://github.com/apache/flink/blob/master/flink-connectors/pom.xml#L83

Reply:
Sure, thanks for clarification, removing the profile